kafka-client/src/main/java/dev/responsive/kafka/store/ResponsivePartitionedStore.java

```diff
@@ -234,17 +234,15 @@ public Position getPosition() {
   public long approximateNumEntries() {
     return partitioner
         .all(partition)
-        .mapToLong(p -> schema.getClient().count(name.cassandraName(), p))
+        .mapToLong(p -> schema.cassandraClient().count(name.cassandraName(), p))
         .sum();
   }
 
   @Override
   public void close() {
+    // no need to flush the buffer here, will happen through the kafka client commit as usual
     if (storeRegistry != null) {
       storeRegistry.deregisterStore(registration);
     }
-    // the client is shared across state stores, so only the
-    // buffer needs to be flushed
-    flush();
```

Comment:
just some minor cleanup, the comment and `flush` call are out of date (the flush invoked here is now a no-op)